internal/driver/glfw/window.go

```diff
@@ -507,10 +507,10 @@ func (w *window) mouseMoved(viewport *glfw.Window, xpos float64, ypos float64) {
 
 	cursor := defaultCursor
 	obj, pos := w.findObjectAtPositionMatching(w.canvas, w.mousePos, func(object fyne.CanvasObject) bool {
-		if wid, ok := object.(*widget.Entry); ok {
-			if !wid.Disabled() {
-				cursor = entryCursor
-			}
+		entry, entOk := object.(*widget.Entry)
+		_, selOk := object.(*widget.SelectEntry)
+		if (entOk && !entry.Disabled()) || selOk {
```

Comment:
This ignores the disabled state of the select entry

Reply:
The disabled state was already ignored before now. Could it be better to exempt the disabled state? I think it makes sense.

Reply:
I'm not sure what you mean - the test above checks the disabled state of Entry - it only sets the bar cursor if the entry is not disabled.
The same should surely hold true for the selectEntry?
It seems fairly standard to not indicate an edit cursor over items that cannot be edited.